admob-lib/AdMob.cs

```diff
@@ -0,0 +1,251 @@
+using Godot;
+using System;
+
+///AUTHOR: Alexander Jungert - 28/08/2020
+public class AdMob : Node 
+    {
+    #region ClassEnums
+    public enum AdResponse { Closed, Opened, Loaded, Started, Finished, Error, Rewarded }
+    #endregion
+    
+    #region Signals
+    [Signal] public delegate void AdCallback(AdResponse response);

```

Comment:
This Signal is not from the original GDScript version, why do you need that? Also is not in Godot Style. 
**Signals
Use the past tense to name signals**
https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html#signals